docs/redirects.asciidoc

```diff
@@ -0,0 +1,9 @@
+["appendix",role="exclude",id="redirects"]
+= Deleted pages
+
+The following pages have moved or been deleted.
+
+[role="exclude",id="log-correlation"]
```

Comment:
@bmorelli25 you mentioned some tooling to handle those redirects, is there a way to express the fact that we want to rename `log-correlation-ids` to `log-correlation` once we can remove this redirect ?

Reply:
Not an official way. I would open an issue and add both id changes to the issue.

Removing old IDs from agent books is tough as sometimes the content can be linked to from 6.x versions of the stack and the link must be removed in all versions between current and 6.x 😨